Automattic/jetpack
# Fix admin menu tests on WP.com

See p9dueE-3LL-p2#comment-6667




#### Changes proposed in this Pull Request:

* add_user_menu test now accounts for the missing `create_users` capability on Dotcom
* Dashboard switcher tracking now expects a blog_id on wpcom and only conditionally has expectations about `record_user_event` being called
* Site card menu class no longer looks for an Atomic site icon when testing WP.com

#### Jetpack product discussion


p9dueE-3LL-p2#comment-6667

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






Automated tests in this repo should pass.

On WP.com:
* Apply these patches to your sandbox: D70265-code, D70207-code
* Run `PHPUnit --testsuite master bar` from the `/bin/tests/jetpack-unit` directory on your wpcom sandbox
